www/js/reset_password.js

```diff
@@ -1,10 +1,17 @@
-$$(document).on('page:init', '.page[data-name="reset-password"]', function (page) {
+$$(document).on('page:init', '.page[data-name="reset-password"]', function (pageEvent) {
+  const page = pageEvent.detail;
```

Comment:
We no longer get a `page` in v2, only a "page event" 😞 
So this needs to be extracted if we want to use it

Reply:
I changed the parameter to be only e in some files, but I forgot about it ^^